src/OrbitGl/include/OrbitGl/TimeGraph.h

```diff
@@ -200,6 +204,7 @@ class TimeGraph : public orbit_gl::CaptureViewElement, public orbit_gl::Timeline
 
   std::shared_ptr<orbit_gl::GlSlider> horizontal_slider_;
   std::shared_ptr<orbit_gl::GlSlider> vertical_slider_;
+  std::shared_ptr<orbit_gl::VerticalSizer> track_header_sizer_;
```

Comment:
Why does it need to be a shared pointer?

Reply:
I think because the picking system expects it.

Reply:
Ah I see. There it probably also is added. Missed that. Thanks

Reply:
Yeah, this is needed due to picking, we may want to clean that up...